main/index.js

```diff
@@ -135,6 +136,10 @@ const ctx = {
   openReleaseNotes: () => { throw new Error('never get here') },
   getUpdaterStatus: () => { throw new Error('never get here') },
   browseTransactionTracker: (/** @type {string} */ transactionHash) => { shell.openExternal(`https://beryx.zondax.ch/v1/search/fil/mainnet/address/${transactionHash}`) },
+  openExternalURL: (/** @type {string} */ url) => {
+    validateExternalURL(url)
+    shell.openExternal(url)
+  },
   showTermsOfService: () => { shell.openExternal('https://pl-strflt.notion.site/Station-Terms-Conditions-e97da76bb89f49e280c2897aebe4c41f?pvs=4') },
```

Comment:
as part of this PR, could you please also remove the other custom URL handlers, to clean up the code base? Also makes the diff or this PR look cooler 😎

Reply:
Good idea 👍